scripts/publish-charts.sh

```diff
@@ -7,22 +7,23 @@ TOOLS_DIR="${BUILD_DIR}/tools"
 STABLE="${GIT_REPO_ROOT}/stable"
 PACKAGE_DIR="${GIT_REPO_ROOT}/build"
 export PATH="${TOOLS_DIR}:${PATH}"
+VERSION="$(git describe --tags --always --dirty)"
```

Comment:
Do we need `--dirty` here? Is this to facilitate local testing?

Reply:
we don't need it, I've removed, it may get in the way if there are other mods (although there really shouldn't be). Went ahead and removed though since we didn't have it before.